scripts/just.js

```diff
@@ -6,13 +10,11 @@ function just(cmd) {
     startIndex = 2;
   }
 
-  process.argv = [
-    ...process.argv.slice(0, startIndex),
-    ...(cmd ? [cmd] : []),
-    '--config',
-    path.resolve(__dirname, 'just-task.js'),
-    ...process.argv.slice(startIndex)
-  ];
+  const config = fs.existsSync(path.resolve(process.cwd(), 'just-task.js'))
+    ? path.resolve(process.cwd(), 'just-task.js')
+    : path.resolve(__dirname, 'just-task.js');
+
+  process.argv = [...process.argv.slice(0, startIndex), ...(cmd ? [cmd] : []), '--config', config, ...process.argv.slice(startIndex)];
```

Comment:
I hate how prettier puts this on one line...would you mind putting it back on multiple lines and adding a `// prettier-ignore` comment inside the opening square bracket to prevent reformatting?